recipes/songbird/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set version = "0.8.2" %}
+{% set name = "songbird" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/cf/44/7dd079ad0b93980e0233ffb03a9941605b4573de535ec4d3a62a212faddc/songbird-0.8.2.tar.gz
+  sha256: 1e152e5d6114135d0299a362ffd082ca525a6b4f8fc0cc1891ea131892e10a37
+
+
+build:
+  number: 0
+  skip: true  # [win]
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+  entry_points:
+    - qiime2.plugins = songbird.q2.plugin_setup:plugin
+
+requirements:
+  host:
+    - python <3.7
+    - pip
+    - numpy >=1.13.1
+
+  run:
+    - python <3.7
+    - ipython
+    - pandas
+    - click
+    - tensorflow >1.7
+    - numpy >1.13
+    - scikit-bio >0.5.4
+    - nose >=1.3.7
+    - seaborn
+    - patsy
+    - scikit-learn
+    - tqdm
+    - h5py
+    - biom-format
+
+test:
+  imports:
+    - songbird
+
+
+about:
+  home: https://github.com/biocore/songbird
+  license: BSD 3-Clause
+  license_family: BSD
```

Comment:
Are we still missing the license_family @mwcraig ?

Reply:
This is good